docs/stagenet.md

```diff
@@ -121,24 +139,19 @@ If all goes well, you should see the node execute the swap protocol. If the swap
 > Note: optionally, you can add the `--transfer-back` flag when starting `swapd` to automatically transfer received XMR back into your original wallet, if you have one opened on the endpoint when starting `swapd`.
 
 ## Maker
-
-1. Create a stagenet wallet:
-```bash
-curl http://localhost:18083/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"create_wallet","params":{"filename":"stagenet-wallet","password":"","language":"English"}}' -H 'Content-Type: application/json'
-```
  
-2. Find your stagenet address:
+1. Find your stagenet address:
 ```bash
 curl http://localhost:18083/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"get_accounts","params":{}}' -H 'Content-Type: application/json'
```

Comment:
That line is accessing `monero-wallet-rpc` directly.  I'll update it to use `swapcli balances` before merging.